site/docs/numbers/porting/lnpChecker.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: lnpChecker
+title: Check number(s) portability
+slug: /numbers/guides/porting/lnpChecker
+description: How to check number(s) portability
+keywords:
+  - bandwidth
+  - numbers
+  - porting
+  - portin
+  - port-in
+  - port
+  - lnp
+  - portability
+  - checker
+  - check
+image: ../../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+A number or set of numbers can be checked to see if they can be ported into the Bandwidth Network.
+:::info
+As an alternative to individual porting the `/bulkPortins` API eliminates the need for the `/lnpchecker` API by sorting the list of TNs automatically into a set of port-in requests by Port Type. For more details follow the [How to Bulk Port](/docs/numbers/guides/porting/createBulkPortins).
+:::
+
+TNs can be ported in a single `/portins` request if all the following are true:
+
+1. They all have the same Port Type (see the `Port Types` section below)
+2. Telephone numbers all have the same losing carrier (toll free telephone numbers all have the same losing RespOrg)
+3. They are all associated with the same billing TN and Service Address
+
+:::caution
+There are also a number of reasons why a TN may not be able to be ported in:
+
+* The TN is in a rate center that is not supported by Bandwidth or any of Bandwidth's partners.
+* The TN is an off-net TN, and the account is configured to support tier zero (on-net) only.
+* The TN belongs to a losing carrier that Bandwidth does not have a Trading Partner Agreement with.
+* The TN is already being processed in another active port-in order.
+* The Bandwidth account has not been enabled for porting.
+Otherwise, the user must separate the TNs into individual `/portins` requests.
+:::
+
+### Port Types
+
+Port Type - is a categorization of how the TNs submitted to the /lnpchecker API will be handled by Bandwidth.
```

Comment:
```suggestion
Port Type is a categorization of how the TNs submitted to the /lnpchecker API will be handled by Bandwidth. 
```

Reply:
Should this be in the create port-in section?

Reply:
Not sure, I put it here because definition references the lnpChecker api rather then porting. Also the table which describes port types is also in the same lnpChecker article